plugin/src/main/scala/org/flywaydb/play/PlayInitializer.scala

```diff
@@ -113,6 +113,9 @@ class PlayInitializer @Inject() (implicit app: Application, webCommands: WebComm
     webCommands.addHandler(flywayWebCommand)
 
     for (dbName <- allDatabaseNames) {
+      if (flywayConfigurations(dbName).cleanOnStart) {
```

Comment:
@vjkoskela is unavailable for the next month.

I disagree with this.  In Play, the difference between "test" and "production" is the way the application is started.  We run many Play apps in a test environment using the "production" way of starting the application.  These include our long-running end-to-end tests, our user test environment, and our pre-production staging environment.  Any one of these should be able to clean the schema, even though it is "production" from a Play point of view.

As a reference to where the built-in modules do it, take a look at the applyDownEvolutions configuration.  While the more typical (and safer) applyEvolutions is generally used in production systems (it won't do any down evolutions), you can specify that you want potentially destructive down evolutions to run if you set applyDownEvolutions.

I would love to give the user the power to do what they need to do, even if it could be misused.